src/frontend/Komponenter/Behandling/Inngangsvilkår/NyttBarnSammePartner/AnnenForelderTidligereVedtaksperioder.tsx

```diff
@@ -55,44 +58,78 @@ const jaNeiMedToolTip = (tidligereVedtak: ITidligereInnvilgetVedtak | undefined)
         harTidligereOvergangsstønad || harTidligereBarnetilsyn || harTidligereSkolepenger;
     return (
         <Tooltip
-            content={`OS: ${tidligereVedtak.harTidligereOvergangsstønad} BT: ${tidligereVedtak.harTidligereBarnetilsyn} SP: ${tidligereVedtak.harTidligereSkolepenger}`}
+            content={`OS: ${harTidligereOvergangsstønad} BT: ${harTidligereBarnetilsyn} SP: ${harTidligereSkolepenger}`}
         >
             <Normaltekst>{mapTrueFalse(harTidligereVedtak)}</Normaltekst>
         </Tooltip>
     );
 };
 
-const AnnenForelderTidligereVedtaksperioder: FC<{
-    registergrunnlagNyttBarn: RegistergrunnlagNyttBarn[];
-}> = ({ registergrunnlagNyttBarn }) => {
-    const andreForeldrer = mapAndreForeldrerMedTidligereVedaksperioder(registergrunnlagNyttBarn);
-
-    if (andreForeldrer.length === 0) {
-        return null;
+const mapSøker = (
+    personopplysninger: IPersonopplysninger,
+    tidligereVedtaksperioder: ITidligereVedtaksperioder
+): AnnenForelderTidligereVedtaksperioder | undefined => {
```

Comment:
Får følelsen av at vi finner "AnnenForelders tidligere vedtaksperioder" her, men er dette egentlig søkerens tidligere vedtaksperioder? 

Er det isåfall mer riktig at objektet heter `TidligereVedtaksperioderPåPartISak` eller noe i den duren - ikke `annenForelder` ?

Reply:
Ja, takk 👍 Skal rydde opp